src/codec_aom.c

```diff
@@ -512,12 +512,16 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
 #endif
         int aomCpuUsed = -1;
         if (encoder->speed != AVIF_SPEED_DEFAULT) {
-            if (encoder->speed < 8) {
-                aomCpuUsed = AVIF_CLAMP(encoder->speed, 0, 6);
-            } else {
+            aomCpuUsed = AVIF_CLAMP(encoder->speed, 0, 9);
+#if defined(AOM_USAGE_ALL_INTRA)
```

Comment:
I will do this, but please note that before this change ALL_INTRA mode was used regardless of ABI version, only if ALL_INTRA was defined...

Reply:
Done

Reply:
> please note that before this change ALL_INTRA mode was used regardless of ABI version, only if ALL_INTRA was defined...

Before this change, ALL_INTRA mode was used for speeds 0-6 only. ALL_INTRA has always supported speeds 0-6 because it was created by forking the GOOD mode.

Speeds 7-9 were added to ALL_INTRA later. That's why we need to check if speeds 7-9 are supported by ALL_INTRA.